core/src/main/scala/org/allenai/common/FileUtils.scala

```diff
@@ -0,0 +1,68 @@
+package org.allenai.common
+
+import au.com.bytecode.opencsv.CSVReader
+
+import java.io._
```

Comment:
At one point yesterday there were too many to explicitly specify, but now seems OK with only 5 classes.  Changing.